tests/ballooneditorui.js

```diff
@@ -134,8 +125,8 @@ describe( 'BalloonEditorUI', () => {
 	} );
 
 	describe( 'destroy()', () => {
-		it( 'destroys the #view', () => {
-			const spy = sinon.spy( view, 'destroy' );
+		it( 'calls EditorUI#destroy', () => {
```

Comment:
Since we're sure this is an instance of `EditorUI`, checking that any longer makes sense IMO.

Reply:
Fixed here and in the all related PR's.